### PR TITLE
feat: HDH-78 web dev tiles & page template for docs landing pages (cd as the example)

### PR DESCRIPTION
**Pull request by wei-harness**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [x] *Optional* Screen Shoot. 
Screen Shot 2023-02-20 at 9 57 47 PM

### CODE REVIEW

**Comment by wei-harness** [comment]
@ravilach There is only the CD landing page for preview the layout and implementation,
we'll need to update the content before we merge the PR.
the content data is defined in /src/components/Docs/{ModuleName}Data.ts

**Comment by kat-enos** [comment]
I think it's a better look if you don't add "Documentation" to the title for each of them, FYI. IMO, you don't need it. I don't usually it on start/technology pages. Plus, it limits what we link to. And if you do, we are using sentence casing and not title caps--in other words, "Documentation" should not be initial-capped. Of course this isn't my page, but in case it's not possible to change the titles once there are in place you might want consider these issues.

**Comment by wei-harness** [comment]
> I think it's a better look (and a shorter heading) if you don't add "Documentation" to the title for each of them, FYI. IMO, you don't need it. I don't usually see it on start/technology pages. Plus, it limits what we link to. And if you do, we are using sentence casing and not title caps--in other words, "Documentation" should not be initial-capped. Of course this isn't my page, but in case it's not possible to change the titles once they are in place you might want consider these issues.

Thanks for the feedback, Kat,
The title and other content of this page could be freely changed by our tech writers and PM, I'll change the title to sentence casing, feel free to make any corrections to the sample page before we merge the PR.

**Comment by ravilach** [comment]
Really likes how this looks! Let's rope in @michaelcretzman and chat with him, as our first new custom page. Will start a convo on slack.

**Review by ravilach — APPROVED** [review]

**Comment by ravilach** [comment]
Updated the PR to just have the tiles in the existing page [https://developer.harness.io/docs/continuous-delivery]

image

CC @KimberlyFields @michaelcretzman

**Comment by michaelcretzman** [comment]
@ravilach this is fine for now. no notes.

File: src/components/Docs/ContinuousDeliveryData.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    {
+      name: "Documentation Topics",
+      description:
+        "These are prudent topics to Continuous Delivery.",
+      list: [
+        {
+          title: "Onboard with Continuous Delivery",
+          module: "cd",
+          description:
+            "This topic covers onboarding with Continuous Delivery.",
```
Comment (michaelcretzman) [inline src/components/Docs/ContinuousDeliveryData.ts:43]:
```suggestion
            "Learn the basics of Harness Continuous Delivery.",
```

File: src/components/Docs/ContinuousDeliveryData.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+          link: "/docs/category/onboard-with-continuous-delivery",
+        },
+        {
+          title: "Services",
+          module: "cd",
+          description:
+            "This topic covers Services.",
```
Comment (michaelcretzman) [inline src/components/Docs/ContinuousDeliveryData.ts:50]:
```suggestion
            "Use Harness services to define your microservices and other workloads.",
```

File: src/components/Docs/ContinuousDeliveryData.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+          link: "/docs/category/services",
+        },
+        {
+          title: "Infrastructure",
+          module: "cd",
+          description:
+            "This topic covers Infrastructure.",
```
Comment (michaelcretzman) [inline src/components/Docs/ContinuousDeliveryData.ts:57]:
```suggestion
            "Define the target infrastructures for your deployments.",
```

File: src/components/Docs/ContinuousDeliveryData.ts [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+          link: "/docs/category/infrastructure",
+        },
+        {
+          title: "Execution",
+          module: "cd",
+          description:
+            "This topic covers Execution.",
```
Comment (michaelcretzman) [inline src/components/Docs/ContinuousDeliveryData.ts:64]:
```suggestion
            "Use built-in steps to model your release process.",
```

File: src/components/Docs/ContinuousDeliveryData.ts [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+          link: "/docs/category/execution",
+        },
+        {
+          title: "Deployments",
+          module: "cd",
+          description:
+            "This topic covers Deployments.",
```
Comment (michaelcretzman) [inline src/components/Docs/ContinuousDeliveryData.ts:71]:
```suggestion
            "Manage how deployments impact your resources.",
```

File: src/components/Docs/ContinuousDeliveryData.ts [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+          link: "/docs/category/deployments",
+        },
+        {
+          title: "GitOps",
+          module: "cd",
+          description:
+            "This topic covers GitOps.",
```
Comment (michaelcretzman) [inline src/components/Docs/ContinuousDeliveryData.ts:78]:
```suggestion
            "Use Harness GitOps to perform Ops tasks.",
```

File: src/components/Docs/ContinuousDeliveryData.ts [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+          link: "/docs/category/gitops",
+        },
+        {
+          title: "Advanced CD",
+          module: "cd",
+          description:
+            "This topic covers Advanced CD.",
```
Comment (michaelcretzman) [inline src/components/Docs/ContinuousDeliveryData.ts:85]:
```suggestion
            "In depth topics to take your CD to the next level.",
```

File: src/components/Docs/ContinuousDeliveryData.ts [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+          link: "/docs/category/advanced-cd",
+        },
+        {
+          title: "CD Dashboards",
+          module: "cd",
+          description:
+            "This topic covers CD Dashboards.",
```
Comment (michaelcretzman) [inline src/components/Docs/ContinuousDeliveryData.ts:92]:
```suggestion
            "Use DORA and other advanced metrics for deployments.",
```

File: src/components/Docs/ContinuousDeliveryData.ts [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+          link: "/docs/category/cd-dashboards",
+        },
+        {
+          title: "CD Tech Reference",
+          module: "cd",
+          description:
+            "This topic covers CD Tech References.",
```
Comment (michaelcretzman) [inline src/components/Docs/ContinuousDeliveryData.ts:99]:
```suggestion
            "Quickly reference settings, permissions, etc.",
```

**Review by michaelcretzman — APPROVED** [review]
approved with the assumption that you'll consider my suggestions.

**Pull request merged**